site/content/blog/2021-11-01-whats-new-in-svelte-november-2021.md

```diff
@@ -0,0 +1,85 @@
+
+
+---
+
+## Community Showcase
+
+**Apps & Sites**
+- [Tangent](http://tangentnotes.com/) is a clean and powerful notes app for Mac & Windows
+- [The Pudding](https://pudding.cool/) is a digital publication that explains ideas debated in culture with visual essays - rebuilt in SvelteKit
+- [Power Switcher](https://powerswitcher.axpo.com/) is an interactive overview of the development of the power supply in Switzerland, as energy sources migrate to cleaner sources
+- [Sublive](https://sub.live/) is a new way of making music by connecting musicians all over the world with a low-latency and high-quality audio network.
+- [Vibify](https://www.vibify.me/) helps you find hidden playlists within your music using your Spotify listening history
+- [Browse Marvel Unlimited by Year](https://marvel.geoffrich.net/) is a SvelteKit site to see what issues are available on Marvel Unlimited for a given year
+- [Files](https://files.community/), a modern file explorer for Windows, has a new site rebuilt with SvelteKit
+- [lil-hash](https://github.com/jackbow/lil-hash) is a simple URL shortener that produces easily rememberable and speakable shortened URLs
+- [PWA Haven](https://github.com/ThaUnknown/pwa-haven) is a collection of small, fast, simple PWA's to replace native OS apps
+- [DottoBit](https://dottobit.com/) is a multi-color 16-bit drawing program with URL sharing built-in
+- [Former Fast Document for Print](https://github.com/zummon/former) is an invoice generator with beautiful designs, abilities for international languages and auto calculation
+
+**Looking for a Svelte project to work on? Interested in helping make Svelte's presence on the web better?** Check out [the list of open issues](https://github.com/svelte-society/sveltesociety-2021/issues) if you'd like to contribute to the Svelte Society rewrite in SvelteKit.
+
+**Podcasts Featuring Svelte**
+- [Svelte Radio](https://www.svelteradio.com/episodes/svelte-summit-is-coming-up-and-svelte-is-growing) dives into the tech behind the recently released Svelte Summit website and a bunch of other fun stuff!
+- [PodRocket](https://podrocket.logrocket.com/rich-harris), LogRocket's podcast, talks Svelte with Rich Harris
+- [Web Rush](https://webrush.io/episodes/episode-153-single-page-application-vs-multi-page-application-with-rich-harris) and Rich Harris talk about the differences between a SPA and MPA, what role the server rendering plays, what client side hydration is, and the state of modern tooling for developing SPA or MPA
+- [devtools.fm](https://devtools.fm/episode/15) talks with Rich Harris about developing engaging data visualizations and building the tools of tomorrow
+- [Helvetikon](https://github.com/noahsalvi/helvetikon) is a community maintained dictionary for the swiss german language
+- [Palitra App](https://palitra.app/) is a search-based color pallete generator
```

Comment:
These two are no podcast, should these be moved to the section below?

Reply:
Oops! Those are in the wrong section